### DISCUSSION TITLE
Help! I'm New. How to show the Text Box

### DISCUSSION THREAD

**Discussion by lemoboy** [fishaudio/fish-speech#410] [Q&A] [answered]
Hi, I'm new here,
How to show the Text Box? I see different layout on youtube and the chinese video? 
What is the wrong with my setup? I have this layout. Thanks
![image](https://github.com/user-attachments/assets/a8e5cf58-e9bf-4c81-9d15-47e848d2b548)

**Comment by greek12man** [comment, marked as answer]
I had the same problem. Solution:
Go to your installation directory, for example fish-speech-main
find the file run_cmd.bat and open it
paste the text

python -m tools.webui

enter, then you will get an address to open it in the browser.

**Reply by lemoboy** [reply]
Thank you so much
